### ISSUE TITLE
Manual updating of longitude and latitude *question*

### ISSUE THREAD

**Issue by mstudi** [oscarmcm/django-places#27]
### DESCRIPTION
This is only a question, not a bug. 

Have you thought of making the longitude and latitude fields update the pin location on the map when they are manually updated and then blurred/tabbed out of those fields? Right now, if one manually updates those fields, the pin is not updated in the map, but still the record will be saved with the new longitude and/or latitude. When the record is reopened, the map is now positioned according to those updates not what was last displayed in the map.

I would be happy to implement this and contribute if interested in this enhancement.

**Comment by oscarmcm** [comment]
@mstudi Yeah, absolutely its a good feature that we should support, so we have a more globalized experience, if you want to tackle down this request, feel free to open a PR! 

```js
var lat = parseFloat(document.getElementById('markerLat').value);
var lng = parseFloat(document.getElementById('markerLng').value);
var newLatLng = new google.maps.LatLng(lat, lng);
marker.setPosition(newLatLng);
```
something like that ☝️ with a few event listeners into the inputs https://github.com/oscarmcm/django-places/blob/master/places/static/places/places.js#L2-L3

**Comment by mstudi** [comment]
No problem. What process do you want me to follow.... Do you want to create another branch or.....?

**Comment by oscarmcm** [comment]
@mstudi yeah, please do this on a new branch and open a PR. Thanks!
